modules/cij/runner.py

```diff
@@ -467,10 +484,17 @@ def hooks_setup(trun: TestRun, parent, hnames=None) -> Dict[str, List[Hook]]:
 def trun_to_file(trun: TestRun, fpath=None):
     """Dump the given trun to file"""
 
+    def dict_factory(instance):
+        """huha"""
+
+        return dict(
+            (x[0], vars(x[1])) if x[0] == "args" else x for x in instance
+        )
```

Comment:
I think we can give a more precise docstring than this one :D

Reply:
Hah, true. Fixed.